src/functionalTest/groovy/com/autonomousapps/jvm/projects/RegexExclusionsProject.groovy

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2024. Tony Robalik.
+// SPDX-License-Identifier: Apache-2.0
+package com.autonomousapps.jvm.projects
+
+import com.autonomousapps.AbstractProject
+import com.autonomousapps.kit.GradleProject
+import com.autonomousapps.kit.Source
+import com.autonomousapps.kit.SourceType
+import com.autonomousapps.model.ProjectAdvice
+
+import static com.autonomousapps.AdviceHelper.actualProjectAdvice
+import static com.autonomousapps.kit.gradle.Dependency.project
+
+final class RegexExclusionsProject extends AbstractProject {
+
+  final GradleProject gradleProject
+
+  RegexExclusionsProject() {
+    this.gradleProject = build()
+  }
+
+  private GradleProject build() {
+    return newGradleProjectBuilder()
+      .withSubproject(':proj') { s ->
+        s.sources = [
+          new Source(
+            SourceType.JAVA, 'Main', 'com/example',
+            """\
+            package com.example.internal;
```

Comment:
The package on line 27 doesn't match the package on line 29.

Note that the builder no longer requires explicitly specifying the package. E.g.:
```groovy
Source.java(
"""\
  package com.example.internal;

  public class Main { ... etc...
""".stripIndent()
).build()
```

Reply:
I tried `Source.java` and started having weird issues with the type of `Builder.sources` not matching anymore, so I just fixed the package declaration instead

Reply:
You may have forgotten to use `.build()`